### PR TITLE
feat(artwork): Add sprite for Mug

### PR DESCRIPTION
**Pull request by Anarchist2**
This PR adds a sprite for the mug purchased in the Deep gift store. To get around being unable to change the art on the mug to suit the current spaceport/planet, the insignia of the Deep is used instead, with the text being updated accordingly.
TWO DAYS REMAIN

## Related Issues
#8801 

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets) with the necessary image, blend, and texture assets: https://github.com/endless-sky/endless-sky-assets/pull/151
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/383

### CODE REVIEW

**Comment by RisingLeaf** [comment]
~why not~
ezgif com-gif-maker_1

File: data/human/culture conversations.txt [hunk 1]
```diff
@@ -89,6 +89,14 @@ mission "Gift Store Interaction"
...
 
 
+outfit "Mug"
+	category "Hand to Hand"
+	cost 50
+	thumbnail "outfit/mug"
+	"capture attack" .1
+	"unplunderable" 1
+	description "You can throw it in a pinch, you guess."
+
```
Comment (bene-dictator) [inline data/human/culture conversations.txt:99]: Looks like this got moved back from outfits.txt.

File: data/human/culture conversations.txt [hunk 2]
```diff
@@ -89,7 +89,6 @@ mission "Gift Store Interaction"
 				decline
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:91]:
```suggestion


```

**Review by lumbar527 — APPROVED** [review]
Looks a lot better than mine.

**Pull request merged**